foundations/installations/computer_basics.md

```diff
@@ -0,0 +1,42 @@
+### Introduction
+
+Needless to say, whilst learning to code, you'll be spending most of your time on a computer. Hence, understanding to work with your computer is crucial. In this lesson, we'll introduce a resource which will quickly get you up to speed with computer basics. 
+
+### Lesson Overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+* Understand computer software and hardware
+* Able to take screenshots
+* Importance of passwords
+
+### Assignment
+
+<div class="lesson-content__panel" markdown="1">
+
+Goodwill Community Foundation resources are excellent to get you acquainted with the basics. They have accompanying videos and illustrations and are easy to read. The following readings should be sufficient, although you are welcome to explore more of GCF's resources.
```

Comment:
I'd add `(GCF)` after "Goodwill Community Foundation" so that the acronym is sort of linked to the full name.